examples/extract_image_feature.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+"""Basic example which iterates through the tasks specified and prints them out.
```

Comment:
Oh, the image feature extraction stuff is still in discuss with Alex. I'll close this pull request for now.